src/components/ItemTooltip/index.jsx

```diff
@@ -119,8 +119,8 @@ const HeaderText = styled.div`
 `;
 
 const ResourceIcon = styled.img`
-  width: 16px;
-  height: 16px;
+  width: 16px !important;
```

Comment:
can we achieve this without !important?

Reply:
It should probably be possible, I can look into it.